tests/test_billing_context_rules.py

```
@@ -0,0 +1,77 @@
+    context, reason, confidence = resolve_work_context(
+        assignment="Neuron Installation",
+        task_text="",
+        work_date=date(2026, 4, 25),
+        start_time=time(9, 0),
+        end_time=time(17, 0),
+    )
+    assert context == "Deployment Support"
+
+
+def test_evening_rule_not_logistics():
+    context, reason, confidence = resolve_work_context(
```

Ruff (0.15.14)

[warning] Unpacked variable `reason` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)